pkgs/tools/inputmethods/fcitx5/update.py

```diff
@@ -12,8 +12,7 @@
 OWNER = "fcitx"
 
 def get_latest_tag(repo, owner=OWNER):
-    r = requests.get( 'https://api.github.com/repos/{}/{}/tags'.format(owner,repo)
-                    , auth=('poscat', 'db5e6fd16d0eb8c36385d3d944e058a1178b4265'))
```

Comment:
Did you revoke that token?

Reply:
It seems that GitHub scans commits for API tokens and revokes those found :facepalm:. The reason I added the token initially is that I was hitting the rate limit (due to debugging). Now that I no longer need to debug the script, I shouldn't be hitting the rate limit anymore.